.github/workflows/ci.yml

```diff
@@ -0,0 +1,75 @@
+on:
+  - push
+  - pull_request
+jobs:
+  test_unix:
+    name: 'Test Unix with CMake'
+    strategy:
+      matrix:
+        os: ['macos-latest', 'ubuntu-latest']
+        config: ['Release', 'Debug']
+        cmake_generator: ['Unix Makefiles', 'Ninja']
```

Comment:
Do we still need to test Ninja?

Reply:
Ninja definitely needs to be supported, but don't think Travis is testing it. I'm running it for all dev, so not critical. A single test build for single configuration on Linux would be nice though.

Somewhat (un)related: we need to make sure these changes work with MSVC 2010 and I don't think that happens with the current Appveyor build. If not, then some merge with a branch off of ci-more, one off tmp build, is needed for smoke testing. Once things work, ci-more will test any future changes before release builds. BTW: I like you suggestion of GHActions cron job for periodic ci-more tests.

Reply:
I just ran `git diff origin/ci-more origin/master`.
The diff is rather long, shouldn't the only difference between these branches be `.travis.yml` and `appveyor.yml`?

Reply:
Correct, but only after master is merge into ci-more. Before that, there is all the ongoing development, since I don't merge that often. Changes apply cleanly because changes only go into master, or the travis/appveyor files on ci-more.

Reply:
I can do a merge now if you want. Hang on ...

Reply:
pushed update. Travis dist updated to focal on ci-more (20.04 LTS). Not sure if it builds yet.

Reply:
You can also test 2017 and 2019 by setting the `APPVEYOR_BUILD_WORKER_IMAGE` environment variable:
https://www.appveyor.com/docs/environment-variables/

Reply:
There is still a small difference between master and ci-more:
`include/flatcc/portable/piovec.h` and `include/flatcc/portable/pversion.h`.

Reply:
huh? need to fix that, thanks.

Reply:
fixed on ci-more

Reply:
I have pushed a script that should (=untested) merge master into ci-more once every week.

Reply:
Great, thanks!

Reply:
I've tested it a bit on a branch of mine and had to do some magic to make it work:

1. create a commit on ci-more with .travis.yml and appveyor.yml for master 
2. merge with master
3. revert .travis.yml and appveyor.yml
4. push ci-more (and trigger .travis.yml and appveyor.yml)

This means that every scheduled run of ci-more will create between 2 and 3 commits. The 2 extra commits are done to avoid merge conflicts around .travis.yml and appveyor.yml..

Reply:
If we remove travis and appveyor from master that won't be an issue, though it is also not a problem as is.
I'd like to completely get rid of travis because they are now changing their domain and the build is broken in strange ways recently. But that requires old compilers on GH Actions on Linux, ci-more.